src/Mod/PartDesign/App/FeatureRevolution.cpp

```diff
@@ -53,13 +53,16 @@ namespace PartDesign {
 
 PROPERTY_SOURCE(PartDesign::Revolution, PartDesign::ProfileBased)
 
+const App::PropertyAngle::Constraints floatAngle = { Base::toDegrees<double>(Precision::Angular()), 360, 1.0 };

```

Comment:
You must be careful with redefinitions of floatAngle in the global namespace as this violates ODR (One Definition Rule). 
Either make them a static variable of the corresponding classes or put them into `namespace {...}` because then the compiler makes sure that different namespaces are generated.

Reply:
Many thanks for this hint!
Indeed, when I comment out the floatAngle definition, the MSVC IDE warns me that it has different definitions and it is unclear what to use. Since it showed me floatAngle definitions an chamfer and polar pattern, I made PR #4563 to fix this as well. With the latest commit to this PR and  #4563 the IDE is silenced.